packages/api/src/routes/helpers/default-error-handler.ts

```diff
@@ -49,12 +54,12 @@ export const errorHandler: ErrorRequestHandler = (err, req, res, next) => {
     capture.error(err, {
       extra: {
         error: err,
-        ...(err instanceof MetriportError ? err.additionalInfo : {}),
+        ...(isMetriportError(err) ? err.additionalInfo : {}),
       },
     });
   }
 
-  if (err instanceof MetriportError) {
+  if (isMetriportError(err)) {
```

Comment:
Bug, b/c `MetriportError` from API and Core are different, depending on the import we used we were not processing errors correctly.

Ticket to move them all to a single place: https://github.com/metriport/metriport-internal/issues/1476